### DISCUSSION TITLE
Lime SDR Mini Crashes

### DISCUSSION THREAD

**Discussion by Jempiken** [AlexandreRouma/SDRPlusPlus#226] [Q&A]
I see Lime SDR mini is possible.. but crashes when starting
Any idea's
Tnx
JP

**Comment by tdbone1** [comment]
is it still like this?

**Reply by SpifTek** [reply]
I have the same problem.
My limesdr mini works with other software and my SDRPlusPlus works with other SDR's
SDR++ recognizes my lime sdr mini but when I hit the play button it give the message that the software is not responding then crashes

**Reply by SpifTek** [reply]
I have tried the setup on 2 different windows 10 PC's

**Reply by SpifTek** [reply]
I am also having the same problem with my hackrf one
